images/win/scripts/SoftwareReport/SoftwareReport.Android.psm1

```diff
@@ -0,0 +1,108 @@
+function Get-AndroidComponentLocation {
+    param(
+        [string] $ComponentName
+    )
+    $path = Join-Path $env:ANDROID_HOME $ComponentName
+    return "Location $path"
```

Comment:
Why do we put everywhere return statement?

Reply:
I believe, it's more clearly.

Reply:
It is habit from other languages like C#, JS.
PowerShell allows to omit `return` word but looks a bit clearer

Reply:
Yep, after that we create something ridiculous:
```
return $packageInfo | ForEach-Object {
        $packageInfoParts = Split-TableRowByColumns $_
        $packageName = $packageInfoParts[0].split(";")[1]
        $packageDescription = $packageInfoParts[2] + ", Revision " + $packageInfoParts[1]
        return Create-AndroidTableObject -PackageName $packageName -Description $packageDescription
    }
```

Reply:
Double return power 😀